easybuild/easyconfigs/l/libpng/libpng-1.6.21-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.21'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('zlib', '1.2.8')]
+
```

Comment:
existing libpng easyconfigs in develop have a custom `sanity_check_paths` here, please include it as well?